flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/DatabaseSync.java

```diff
@@ -41,18 +42,23 @@
 import java.sql.SQLException;
 import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Properties;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 public abstract class DatabaseSync {
     private static final Logger LOG = LoggerFactory.getLogger(DatabaseSync.class);
     private static final String LIGHT_SCHEMA_CHANGE = "light_schema_change";
     private static final String TABLE_NAME_OPTIONS = "table-name";
+    public static final String TABLE_DDL_TO_LOWER_CASE = "1";
+    public static final String TABLE_DDL_TO_UPPER_CASE = "2";
```

Comment:
It seems that the main control here is the field name? Is it better to use Boolean? Because doris fields are not case-sensitive.

Reply:
> 似乎这里的主要控件是字段名称？使用布尔值更好吗？因为多丽丝字段不区分大小写。

Boolean values can be used

Reply:
Maybe add a boolean switch. If it's not case-sensitive, you can convert it all to lowercase. If it's case-sensitive, just leave it as is.

Reply:
> 也许添加一个布尔开关。如果它不区分大小写，则可以将其全部转换为小写。如果区分大小写，则保持原样。

I have figured out how to change everything else, but I found that Boolean cannot express multiple meanings

As you said, either keep it the same or use all lowercase. If you convert it all to uppercase, it won't be easy to achieve

Reply:
Do you mean doris column names are capitalized, or table names are capitalized? Because column names are not case sensitive

Reply:
> Do you mean doris column names are capitalized, or table names are capitalized? Because column names are not case sensitive

What I'm talking about is the case of the created table fields

Reply:
Doris does not distinguish between uppercase and lowercase fields. The field you create is NAME, and you can also query it using name.

Reply:
> Doris不区分大写和小写字段。您创建的字段是 NAME，您也可以使用 name 进行查询。

I understand what you mean. I added this new feature to unify tables and fields, which I believe is also a form of standardization

Reply:
But because the table names of doris are case-sensitive and the fields are not different, if they are forcibly unified, it may not be used in some scenarios? For example, should the table name be in uppercase and the fields in lowercase?

Reply:
> 但是因为 doris 的表名区分大小写，字段没有区别，如果强行统一，可能在某些场景下就不用了呢？例如，表名是否应为大写，字段应为小写？

I have implemented this part of the logic, and any operations on tables and fields are done together

Reply:
What I mean is that these two should not be strongly bound. They can be consistent or inconsistent. There is no need to unify them.

Reply:
> 我的意思是，这两者不应该被强硬地捆绑在一起。它们可以是一致的，也可以是不一致的。没有必要统一它们。

Okay, I can complete this operation by distinguishing between two fields. Let me make some changes.
But multiple state representations cannot be completed through Boolean values